services/common/config/stations_cfg.json

```diff
@@ -5,6 +5,6 @@
     "nsg": "http://127.0.0.1:5103/Files",
     "sgs": "http://127.0.0.1:5104/Files",
     "cadip": "http://127.0.0.1:5000/Files",
-
+    "cadip_session": "http://127.0.0.1:5000/Sessions",
```

Comment:
Do we need also to modify the cluster configuration to add this value ? Do we need an additional pod on the cluster ? Did you discuss it with Nicolas ?

Reply:
No, this file was removed in other branch. https://github.com/RS-PYTHON/rs-server/pull/56